glutin/src/context.rs

```diff
@@ -280,7 +282,9 @@ pub enum GlProfile {
 pub enum ContextApi {
     /// OpenGL Api version that should be used by the context.
     ///
-    /// When using `None` as a `ApiVersion` any OpenGl context will be picked.
+    /// When using `None` as a `ApiVersion` any OpenGl context will be picked,
```

Comment:
```suggestion
    /// When using `None` as `ApiVersion` any OpenGl context will be picked,
```

Same below on the `Gles` item.

Reply:
Also, `OpenGl` or `OpenGL`?